beacon-chain/verification/verification_test.go

```diff
@@ -0,0 +1,15 @@
+package verification
+
+import (
+	"os"
+	"testing"
+
+	"github.com/OffchainLabs/prysm/v6/beacon-chain/blockchain/kzg"
+)
+
+func TestMain(t *testing.M) {
+	if err := kzg.Start(); err != nil {
```

Comment:
You have to flip a power switch on this package? :open_mouth:

Reply:
yeah, im with Kasey on this one. kzg.Start() doesn't even use sync.Once.

https://github.com/OffchainLabs/prysm/blob/v6.0.2/beacon-chain/blockchain/kzg/trusted_setup.go#L27

Can we follow up in another PR? imo this should just be an init function on the kzg package to load the kzgContext such that callers do not need to do this.

Reply:
Yes. It is actually not peerDAS specific, but started with blobs at Deneb.
Adding a new task in the list, for an eventual PR.